modules/prebidServerBidAdapter/index.js

```diff
@@ -918,8 +918,8 @@ Object.assign(ORTB2.prototype, {
         (seatbid.bid || []).forEach(bid => {
           let bidRequest = this.getBidRequest(bid.impid, seatbid.seat);
           if (bidRequest == null) {
-            if (!s2sConfig.allowUnknownBidderCodes) {
-              logWarn(`PBS adapter received bid from unknown bidder (${seatbid.seat}), but 's2sConfig.allowUnknownBidderCodes' is not set. Ignoring bid.`);
+            if (!s2sConfig.allowAlternateBidderCodes) {
```

Comment:
this is a breaking change - and not how I interpret the proposal. However, I realize now that how I'd interpret the proposal is not possible to implement.

Proposal says:

> 3. Update the pbsBidAdapter to look in bidderSettings as well as s2sConfig and enforce allowedAlternateBidderCodes.

I read it as "the pbs adapter should also enforce `bidderSettings[].allowAlternateBiddercodes`, and default to `s2sConfig.allowUnknownBidderCodes`". However, the pbs adapter cannot know what the adapter code is - it could be any of the bidders in the request - so what should it look up in `bidderSettings`? @bretg, @patmmccann what is your interpretation?